src/foam/u2/wizard/common/ReviewWizardletView.js

```diff
@@ -8,7 +8,21 @@ foam.CLASS({
   package: 'foam.u2.wizard.common',
   name: 'ReviewWizardletView',
   extends: 'foam.u2.View',
-    
+
+  messages: [
+    { name: 'TITLE', message: 'Review your Deposit ' }
+  ],
+
+  css: `
+    ^ .generic-container {
+      width: 343px;
```

Comment:
```suggestion
      width: 75%;
```
Can increase or decrease if it is too wide/narrow

Reply:
actually width should be removed - the card will expand to fill the container, which will usually be ample surrounding whitespace in a centered wizard

Reply:
How is that white space set? Does the wizard define how wide the wizardlets will be?